src/security.cs

```diff
@@ -35,13 +35,15 @@ interface SecPolicyIdentifier {
 			
 		[Field ("kSecPolicyAppleIPsec")]
 		NSString AppleIPsec { get; }
-#if MONOMAC
+
+		[NoiOS][NoWatch][NoTV]
```

Comment:
Is Catalyst included in this or not? I'd ideally like to see either:

`[NoiOS][NoMacCatalyst][NoWatch][NoTV]`
or
Mac and MacCatalyst with versions.